invenio_app_ils/ui/src/invenio_app_ils/App.js

```diff
@@ -7,14 +7,20 @@ import {
 } from './routes';
 import history from './history';
 import { NotFound } from './common/components';
+import { AuthenticationGuard, UnAuthorized } from './authentication/components';
 
 export default class App extends Component {
   render() {
     return (
       <Router history={history}>
         <Switch>
           <Route exact path="/" component={FrontSiteRoutes} />
-          <Route path={`${BackOfficeURLS.home}`} component={BackOfficeRoutes} />
+          <AuthenticationGuard
+            path={`${BackOfficeURLS.home}`}
+            authorizedComponent={BackOfficeRoutes}
+            unAuthorizedComponent={UnAuthorized}
+            roles={['admin', 'librarian']}
```

Comment:
should we hardcode the roles? maybe it's better to read them from invenioConfig?

Reply:
I didn't want to add just another config for these 2 variables as we said we will see in the end what we really need to expose from invenio config ( backend ) and consume it. This component is meant to be used also in other places with probably different set of roles needed.